react/Radio/Readme.md

```diff
@@ -2,126 +2,132 @@
 
 ```jsx
 import Radio from 'cozy-ui/transpiled/react/Radio'
+import RadioGroup from 'cozy-ui/transpiled/react/RadioGroup'
+import FormControlLabel from 'cozy-ui/transpiled/react/FormControlLabel'
 import Variants from 'cozy-ui/docs/components/Variants'
+const Box = ({ children }) => {
+  return <div style={{ height: '3rem', width: '3rem', border: '2px dashed #CCC', display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+    { children }
+  </div>
+}
 
-let initialVariants = [
-  { error: false, disabled: false }
+initialState = { radioValue: 'item1' }
+
+const initialVariants = [
+  { checked: false, disabled: false, secondary: false }
 ]
 
-if (isTesting()) {
-  initialVariants = [
-    ...initialVariants,
-    { error: true, disabled: false },
-    { error: false, disabled: true }
-  ]
+const handleChange = event => {
+  setState({ radioValue: event.target.value  })
 }
 
-<Variants initialVariants={initialVariants}>{
-  variant => (
-    <form>
-      <div>
-        <Radio
-          name="radioForm"
-          value="radioValue1"
-          label="This is a radio button"
-          error={variant.error}
-          disabled={variant.disabled}
-        />
-        <Radio
-          name="radioForm"
-          value="radioValue2"
-          label="This is also a radio button"
-          error={variant.error}
-          disabled={variant.disabled}
-        />
+const bboxStyle = { border: '1px solid red' }
+
+;
+
+<>
+  <RadioGroup
+    aria-label="radio"
+    name="radioName"
+    value={state.radioValue.toString()}
+    onChange={handleChange}
+  >
+    <FormControlLabel
+      value="item1"
+      label="This is a radio button"
+      control={
+        <Radio />
+      }
+    />
+    <FormControlLabel
+      value="item2"
+      label="This is an error radio button"
```

Comment:
I was wondering why errored radio button was not red when unselected like for old version.

It fits Figma spec so I have my answer, but I realised that it was not libelled as `error` in figma but as `intent`, which is different than an error in my point of view.

Should we reflect this on the example by removing the word `error`?

Reply:
remplacé par "intent"...